File: android/src/main/java/com/facebook/flipper/plugins/inspector/descriptors/utils/stethocopies/FragmentManagerAccessor.java
/*
 * Copyright (c) Facebook, Inc. and its affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */

package com.facebook.flipper.plugins.inspector.descriptors.utils.stethocopies;

import java.util.List;
import javax.annotation.Nullable;

public interface FragmentManagerAccessor<FRAGMENT_MANAGER, FRAGMENT> {
  @Nullable
  List<FRAGMENT> getAddedFragments(FRAGMENT_MANAGER fragmentManager);
}

File: android/src/main/java/com/facebook/flipper/plugins/uidebugger/stetho/FragmentManagerAccessor.kt
/*
 * Copyright (c) Meta Platforms, Inc. and affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */

package com.facebook.flipper.plugins.uidebugger.stetho

interface FragmentManagerAccessor<FRAGMENT_MANAGER, FRAGMENT> {
  fun getAddedFragments(fragmentManager: FRAGMENT_MANAGER): List<FRAGMENT>
}
